index.html

```diff
@@ -14,46 +14,190 @@
       rel="stylesheet"
     />
 
-    <title>portfolio-setup Project</title>
+    <title>portfolio-setup</title>
   </head>
   <body>
-    <!-- my-logo section -->
-    <header class="toolbar-section">
-      <div class="logo">
-        <a href="">My Logo</a>
-      </div>
-
-      <div>
-        <p><span class="iconify" data-icon="eva:menu-outline"></span></p>
-      </div>
-    </header>
-
-    <!-- main section -->
-    <main class="main-section">
-      <div class="content">
-        <p class="author">
-          I’m Ismail <br />
-          Glad to see <br />
-          you!
```

Comment:
## This part of your project is missing, I would like to advice that you include this part of your project as it is required of you to build this portfolio from one section to another.

![image](https://user-images.githubusercontent.com/77106962/143558743-85940c7a-30d1-41f2-b9c9-3d3e420067f0.png)